app/services/loader.js

```diff
@@ -37,12 +37,13 @@ export default Service.extend({
     data = data || config.data || null;
 
     if (!config.isExternal) {
-      if (!url.startsWith('/')) {
-        url = `/${url}`;
+      if (!url.startsWith('http')) {
+        if (!url.startsWith('/')) {
+          url = `/${url}`;
+        }
+        url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
What is it doing?

Reply:
actually sir problem was here. 
![export explain](https://user-images.githubusercontent.com/72552281/102043154-a9cf0580-3df9-11eb-869f-0bcfb658516e.png)

while download anything like invioce, ticket etc, server send dowload url of this type ----> /v1/orders/123456ased/download.pdf.. loader take this download url and append it after api address and make it kike this -----> https://api.eventyay.con/v1/orders/123456ased/dowload.pdf...

but when we comw to download zip, server give full address like----> https://api.evetyay.com/v1/export/custom/........, and when we use loader.downloadFile it apeend it after api address and make like this--->   https://api.eventyay.com/https://api.eventyay.com/v1/export..... this is why download is nnot working.

to elove this i make a check before appending downloadurl after host api. 
if downloadurl starts with hhtp then no need to append downloadurl after host api, leave it as it is.

that's why i done this. 
sir, if you have better approach suggest me.

Reply:
Simple solution is to use `isExternal: true`

Reply:
Ok sir, i got it. I will do this